libpod/logs/log.go

```diff
@@ -34,6 +34,7 @@ type LogOptions struct {
 	Details    bool
 	Follow     bool
 	Since      time.Time
+	Until      time.Time
```

Comment:
The backend should use this value, otherwise this should be omitted.

Reply:
@Luap99 do you mean in `container_log_linux`? Compat uses this struct so I had to add this in order to do some implementation there as well as in `libpod/container_log.go`

Reply:
Sorry my bad. I missed that you use it for journald already but you also have to implement it for `readFromLogFile` in `libpod/container_log.go`